WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -17,6 +17,8 @@ class NUXAbstractViewController : UIViewController
     let helpBadgeSize = CGSize(width: 12, height: 10)
     let helpButtonContainerFrame = CGRect(x: 0, y: 0, width: 44, height: 44)
 
+    var dismissBlock: ((cancelled: Bool) -> Void)?
```

Comment:
I disagree. I think calling it a completion block implies its called only when the sign in flow is completed.  Since this is a block that will be execute when the view controller is dismissed via `dismissViewControllerAnimated` calling it a dismiss block seems best.